sklearn/utils/_array_api.py

```diff
@@ -590,3 +590,8 @@ def _estimator_with_converted_arrays(estimator, converter):
             attribute = converter(attribute)
         setattr(new_estimator, key, attribute)
     return new_estimator
+
+
+def _array_api_atol(dtype):
```

Comment:
I like having this function but I don't like the name :-/ In particular I don't like having `array_api` in there, seems kinda redundant (if we want it for namespacing reasons then I'd recommend "don't do `from _array_api import atol` and use `_array_api.atol()` instead".)

Some alternatives that I like more: `atol_for(dtype)`, `atol(dtype)`, `atol_for_type(dtype)`, `absolute_tolerance(dtype)`

Maybe we can combine this with other ideas to find a good name?

Reply:
Yes you are right. I like `atol_for_type`. Otherwise, since it is an absolute tolerance used for testing, could we call it something like `testing_atol` or `testing_atol_for_type`?

Reply:
I'd not include `testing` in the name